probe-rs/src/debug/unit_info.rs

```diff
@@ -426,7 +414,7 @@ impl UnitInfo {
                     },
                     // Property of variables that are of DW_TAG_subrange_type.
                     gimli::DW_AT_lower_bound => match attr.value().udata_value() {
-                        Some(lower_bound) => child_variable.range_lower_bound = lower_bound as i64,
+                        Some(bound) => child_variable.range_lower_bound = bound as i64,
```

Comment:
@noppej do you know why these aren't `sdata_value()`?

Reply:
@bugadani Is it viable that DW_AT_lower_bound will be a negative value?

Reply:
Its a long time ago, so my memory might be faulty, but it is probably because i64 conversion is the number format the DAP spec uses, so I think the design of the `Variable` struct 'accidentally' inherited it :)